tools/scripts-core/src/config/webpack.config.js

```diff
@@ -28,7 +28,10 @@ export default async (env = {}) => {
 	// User configuration file
 	const userConfigPath = presetApi.getUserConfig(['webpack', 'config', presetApi.mode]);
 	if (userConfigPath) {
-		const userConfigAbsolutePath = utils.path.getAbsolutePath(userConfigPath);
+		let userConfigAbsolutePath = utils.path.getAbsolutePath(userConfigPath);
+		if (process.platform === 'win32') {
+			userConfigAbsolutePath = `file:///${userConfigAbsolutePath}`;
```

Comment:
as we use `await import(userConfigAbsolutePath)` we need a protocol supported by windows.
I have found on stackoverflow this trick :point_up: and it works